synapse/tests/test_cortex.py

```diff
@@ -4,6 +4,31 @@
 
 class CortexTest(s_test.SynTest):
 
+    def test_splice_sync(self):
+        with self.getTestDmon(mirror='dmoncore') as dst_dmon:
+            name = 'core'
+            host, port = dst_dmon.addr
+            dst_core = dst_dmon.shared.get(name)
+            dst_core_addr = f'tcp://{host}:{port}/{name}'
+
+            # Spin up a source core configured to send splices to dst core
+            conf = {'splice:sync': dst_core_addr}
+            with self.getTestCore(conf=conf) as src_core:
+
+                # Form a node and make sure that it exists
+                with src_core.snap() as snap:
+                    snap.addNode('teststr', 'teehee')
+                    self.nn(snap.getNodeByNdef(('teststr', 'teehee')))
+
+                import time; time.sleep(3)  # FIXME wait on event
```

Comment:
I need to plumb something in here to wait on as the src_core exits before connecting to the dst_core. I could put the dst_core stuff inside the src_core with block, but it would still be a race.

Reply:
We can add a event on the Cortex which we can `fire()` in the loops; and use a `waiter()` for testing.  The overhead of firing the event _should_ be pretty small in production, compared to the network io overhead in many cases.